spec/determinator/control_spec.rb

```diff
@@ -51,5 +52,28 @@ def execute(example)
         end
       end
     end
+
+    context 'single bucketed, half rollout features should be on or off, randomly, for different calls' do
+      let(:feature_name) { 'feature_flags/simple/half_rollout_single.json' }
+      let(:certainty_percent) { 99.999 }
+
+      it 'should randomly give true or false results at different times' do
```

Comment:
This test seems kinda weird in that it doesn't check you got about 50% of each thing. e.g. if you ran 10k iterations then you'd expect _roughly_ 5k of each (+- a safety margin) and ditto for say 10% or 90%. In other words, this doesn't actually check the randomness but just that both results can be produced.

However, the implementation seems fine, so whether you consider this a blocker I'll leave up to you.

Reply:
You're right, it doesn't; let me see if I can find something slightly more suitable.